lib/widgets/clipboard.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:device_info_plus/device_info_plus.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+/// Copies [data] to the clipboard and shows a popup on success.
+///
+/// Must have a [Scaffold] ancestor.
+///
+/// On newer Android the popup is defined and shown by the platform. On older
+/// Android and on iOS, shows a [Snackbar] with [successContent].
+///
+/// In English, the text in [successContent] should be short, should start with
+/// a capital letter, and should have no ending punctuation: "{noun} copied".
+void copyWithPopup({
+  required BuildContext context,
+  required ClipboardData data,
+  required Widget successContent,
+}) async {
+  await Clipboard.setData(data);
+  final deviceInfo = await DeviceInfoPlugin().deviceInfo;
+
+  // Early return on !mounted would be better, but:
+  //   https://github.com/dart-lang/linter/issues/4007
+  if (context.mounted) {
```

Comment:
Here's a fun way to express this, which I developed the other day in my draft for #11 :
```dart
    if (context.mounted) {} // https://github.com/dart-lang/linter/issues/4007
    else {
      return;
    }
```